sdk/python/lib/pulumi/runtime/stack.py

```diff
@@ -311,6 +311,11 @@ def register_stack_transform(t: ResourceTransform):
             "The Pulumi CLI does not support transforms. Please update the Pulumi CLI."
         )
 
+    loop = asyncio.get_event_loop()
+    pending = asyncio.all_tasks()
+    rpcs = {task for task in pending if task._coro.__name__ == "rpc_wrapper"}
```

Comment:
I think it's ok as is for me, but it does seem slightly brittle to depend on the function name here, as it could break when refactoring the function

Maybe we could add a marker attribute the function along with a comment to make it more explicit?

```
async def rpc_wrapper(*args, **kwargs):
  ...
rpc_wrapper.not_sure_what_to_name_this = True
```

and then use that instead of `__name__`.

Or we could have a test, but I don't see how to have a test that's not a flakey-disaster.

Reply:
I didn't realize python could do that.  I cajjed this `is_rpc_wrapper` now, but open to other suggestions. Thanks!

Reply:
I gave this a try, but unfortunately it doesn't seem to work.  No matter what I tried it kept erroring saying something like "coroutine has no attribute `is_rpc_wrapper`", even for the `rpc_wrapper`.

Not sure if I'm just holding it wrong, but I went back to using `__name__` for now.  If anyone has ideas of how to make this work without relying on the name still I'd be happy to hear.